documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+
+|`heartbeat.interval.ms`
+|`0`
+|Controls how frequently heartbeat messages are sent. +
+This property contains an interval in milli-seconds that defines how frequently the connector sends messages into a heartbeat topic.
+This can be used to monitor whether the connector is still receiving change events from the database.
+You also should leverage heartbeat messages in cases where only records in non-captured tables are changed for a longer period of time.
+In such situation the connector would proceed to read the log from the database but never emit any change messages into Kafka,
+which in turn means that no offset updates will be committed to Kafka.
+This may result in more change events to be re-sent after a connector restart.
+Set this parameter to `0` to not send heartbeat messages at all. +
+Disabled by default.
+
+|`heartbeat.topics.prefix`
+|`__debezium-heartbeat`
+|Controls the naming of the topic to which heartbeat messages are sent. +
+The topic is named according to the pattern `<heartbeat.topics.prefix>.<server.name>`.
+
+|`snapshot.delay.ms`
+|
+|An interval in milli-seconds that the connector should wait before taking a snapshot after starting up; +
+Can be used to avoid snapshot interruptions when starting multiple connectors in a cluster, which may cause re-balancing of connectors.
+
+|`snapshot.fetch.size`
+|`2000`
+|Specifies the maximum number of rows that should be read in one go from each table while taking a snapshot.
+The connector will read the table contents in multiple batches of this size. Defaults to 2000.
+
+|`snapshot.lock.timeout.ms`
+|`10000`
+|An integer value that specifies the maximum amount of time (in milliseconds) to wait to obtain table locks when performing a snapshot. If table locks cannot be acquired in this time interval, the snapshot will fail (also see link:#snapshots[snapshots]). +
+When set to `0` the connector will fail immediately when it cannot obtain the lock. Value `-1` indicates infinite waiting.
+
+|`snapshot.select.statement.overrides`
+|
+|Controls which rows from tables will be included in snapshot. +
+This property contains a comma-separated list of fully-qualified tables _(SCHEMA_NAME.TABLE_NAME)_. Select statements for the individual tables are specified in further configuration properties, one for each table, identified by the id `snapshot.select.statement.overrides.[SCHEMA_NAME].[TABLE_NAME]`. The value of those properties is the SELECT statement to use when retrieving data from the specific table during snapshotting. _A possible use case for large append-only tables is setting a specific point where to start (resume) snapshotting, in case a previous snapshotting was interrupted._ +
+*Note*: This setting has impact on snapshots only. Events captured during log reading are not affected by it.
+
+|`source.struct.version`
```

Comment:
That option isn't existing for that connector.

Reply:
Is this connector specific or handled by the core (we thought the latter) ?

Reply:
Done